models/articles.models.js

```diff
@@ -72,3 +72,14 @@ exports.checkArticleExists = (article_id) => {
       }
     });
 };
+
+exports.updateArticleById = (article_id, inc_votes) => {
+  const doesExist = this.checkArticleExists(article_id);
+  const query = db.query(
+    `UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *;`,
+    [inc_votes, article_id]
+  );
+  return Promise.all([doesExist, query]).then((results) => {
```

Comment:
Do you need two database queries here ? What will you get from the `RETURNING *` if the article doesn't exist ?